### PR TITLE
Aslone/cloudbuild docs

### PR DESCRIPTION
**Pull request by ajslone**
Please have a look. This PR is just updated docs for caliban base images.

### CODE REVIEW

File: docs/explore/base_image.rst [hunk 1]
```diff
@@ -10,23 +10,78 @@ environment inside of each container. This guarantees you a truly blank slate;
...
 libraries that will be present. No more version clashes or surprises.
 
-Caliban uses one of two base images, depending on whether you're running in GPU
-(default) or CPU mode:
+Caliban uses a set of base images covering a set of common combinations of
+python and cuda versions. Our current matrix of supported combinations:
 
-* ``gcr.io/blueshift-playground/blueshift:gpu`` for the default GPU mode
-* ``gcr.io/blueshift-playground/blueshift:cpu`` for CPU, or ``--nogpu``\ , mode
++-----------+------------+------------+
```
Comment (sritchie) [inline docs/explore/base_image.rst:16]: can you add a list of the images, or maybe link to the blueshift-playground folder where they live, so the user can figure it out themselves based on the matrix?

File: docs/explore/base_image.rst [hunk 2]
```diff
@@ -10,23 +10,78 @@ environment inside of each container. This guarantees you a truly blank slate;
...
+To manually start a Cloud Build for these docker images, navigate to the top-level
+of the caliban source repository, and use the
+`gcloud builds <https://cloud.google.com/cloud-build/docs/running-builds/start-build-manually#gcloud>`_
+command:
+
+.. code-block:: bash
+
+   gcloud builds submit .
+
+By default this uses your default project and the ``cloudbuild.json`` file in your current
```
Comment (sritchie) [inline docs/explore/base_image.rst:79]: excellent!

File: docs/explore/base_image.rst (hunk 1 above)
Comment (ajslone) [reply docs/explore/base_image.rst:16]: good idea, I will add

**Comment by sritchie** [comment]
🚢

**Pull request merged**